src/us/kbase/narrativejobservice/sdkjobs/SDKMethodRunner.java

```diff
@@ -1065,14 +1070,19 @@ private static void addAweTaskDescription(
 		dbTask.setAppJobId(appJobId);
 		db.insertExecTask(dbTask);
 	}
+
+
 	/**
 	 * Saves state in mongodb to allow the job to communicate its status
-	 * @param ujsJobId ID Generated from UJS
-	 * @param jobId ID Generated from scheduler
-	 * @param jobInput RunJob Params
-	 * @param schedulerType Scheduler type, such as Condor or Awe
-	 * @param config Configuration file
-	 * @return void
+	 * @param ujsJobId (UJS ID)
+	 * @param jobId (CONDOR ID)
```

Comment:
Maybe say SCHDULER ID

Reply:
Changed to /mnt/condor and changed comment